tools/x-repo-types/go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/algorand/go-algorand/tools/x-repo-types
+
+go 1.17
+
+replace github.com/algorand/go-algorand => ../..
```

Comment:
@cce - Got it working. All I really needed was this replace directive.

Reply:
nice!